parm/input.benchmark_ccpp.nml.IN

```diff
@@ -196,6 +196,7 @@
   iopt_stc     = 1
   debug        = .false.
   nstf_name    = 0,0,0,0,0
+  frac_grid    = .false.
```

Comment:
I think it better if this were defined as a variable in default_vars. This is what I had added for the upcoming CMEPS commit which has fractional grid capabilities:

# default settings for fractional grid
export FRAC_GRID='.F.'
export CPLMODE='nems_orig'

Then we'd have frac_grid = @[FRAC_GRID] in the namelists.

Reply:
I think it is good to set frac_grid = @[FRAC_GRID], and then in default_vars.sh, define FRAC_GRID=.false. for this commit.

Reply:
Updates have been made to the following 3 files:
parm/input.benchmark_ccpp.nml.IN
parm/input.mom6_ccpp.nml.IN
tests/default_vars.sh